includes/course-theme/class-sensei-course-theme.php

```diff
@@ -152,7 +153,7 @@ public function override_theme() {
 	public function add_rewrite_rules() {
 		global $wp;
 		$wp->add_query_var( self::QUERY_VAR );
-		add_rewrite_rule( '^' . self::QUERY_VAR . '/([^/]*)/([^/]*)/?\??(.*)', 'index.php?' . self::QUERY_VAR . '=1&post_type=$matches[1]&name=$matches[2]&$matches[3]', 'top' );
+		add_rewrite_rule( '^' . self::QUERY_VAR . '/([^/]+)/([^/]+)(?:/([0-9]+))?\??(.*)', 'index.php?' . self::QUERY_VAR . '=1&post_type=$matches[1]&name=$matches[2]&page=$matches[3]&$matches[4]', 'top' );
```

Comment:
I think we should bump the number in the `sensei_course_theme_query_var_flushed` again after updating new rule, right?

Reply:
Yup, thought we already did but the previous bump was in the 4.0.1 release. [8aee6a8](https://github.com/Automattic/sensei/pull/4686/commits/8aee6a886e4ae352bc84c0928a67d2ec6fd4ef1e)